core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -116,12 +116,12 @@
           MeasurementUnit.create(6, bytes));
 
   // Common histogram bucket boundaries for bytes received/sets DistributionViewDescriptors.
-  private static final List<Double> RPC_BYTES_BUCKET_BOUNDARIES = Arrays.asList(
```

Comment:
The List used here is not immutable, so the ALL_CAPS style should not apply. Either change the name, or use Collections.unmodifiableList. (also other similar uses).